### DISCUSSION TITLE
Startverzögerung CloudMatic

### DISCUSSION THREAD

**Discussion by blackbasket** [OpenCCU/OpenCCU#2254] [🇩🇪 Ideen]
Hallo zusammen,

nachdem es gestern an einem Standort wieder mal einen Stromausfall gab, ist mir folgende Idee gekommen.

Wäre es möglich, dass für den Start von CloudMatic eine Verzögerung (zB 5 Minuten) eingebaut wird?

Hintergrund: Wenn die RM schneller bootet als die restliche Infrastruktur bzw. noch kein WAN verfügbar ist, läuft die Verbindung in einen Fehler, der sich nur durch manuellen Neustart der VPN-Verbindung beheben lässt.

Solange man noch eine eigene VPN-Verbindung in das Netz hat, kann man dies zumindest umsetzen. Sonst kommt man nur noch dran, wenn man physikalisch vor Ort ist.

LG,
Marcel

**Comment by Baxxy13** [comment]
Mach doch einfach per WebUI-Programm (Script, SystemExec() ) 5Min nach Zentralenstart einen Restart von CloudMatic.

**Reply by blackbasket** [reply]
Klar, geht auch... dachte, es wäre ggf für die Allgemeinheit interessant ;)
